pkg/theiactl/commands/root.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "theiactl",
+	Short: "theiactl is the command line tool for Theia",
+	Long: `theiactl is the command line tool for Theia that supports the
+policy recommendation feature`,
```

Comment:
For the time being I'd keep "short" and "long" description the same or change it to

`theiactl is the command line tool for Theia which provides access to some network flow visibility capabilities`

I don't think we want to expand the long description for every feature we add.

Reply:
Thanks, changed.